### ISSUE TITLE
WS2815 Support

### ISSUE THREAD

**Issue by Andy79881** [lizardsystems/yawl-controller#21]
Does the YAWL HW-Support 12V LED Stripes as the WS2815?

**Comment by chunkysteveo** [comment]
Not in it's current state. Most of the capacitors are rated below 12V, so they would blow if you connected a 12V source to it. The Fuse is only rated to 6V. And the level shifter is only tolerant up to 6.5 V. The voltage regulator would work, so that's good...(!)

So in short - No!

If you replaced the components with similar ones that could tolerate and work with 12V, it would work well.

**Comment by Orygin** [comment]
I also am wondering about ws2815 strips.
Would powering the controller with 5v, and the led with 12v work ?
Assuming I connect the grounds of the controller and the strip together

**Comment by chunkysteveo** [comment]
> I also am wondering about ws2815 strips. Would powering the controller with 5v, and the led with 12v work ? Assuming I connect the grounds of the controller and the strip together

Actually yes.. if the 12v is isolated to just the LEDs, and you step it down to 5V before inputting to the controller - it _should_ work. The 12V WS2815 still use 5V logic on the data lines. And yeah, both grounds connected.

**Comment by Orygin** [comment]
Thanks that's perfect.
I have purchased a strip and power supply to test it, I'll report back if I
experience any issues

**Comment by Orygin** [comment]
I can confirm I can drive a WS2815 led strip with this controller. Just power the chip with 5v, the LEDs with 12v and bridge the grounds together

**Comment by chunkysteveo** [comment]
That's awesome to hear! Good to know the theory works as expected! I have some 12V COB WS2812B chips, so may do this method too.

Thanks!

**Comment by numindast** [comment]
This also works fine for 24v strips, as long as you have a source of 5v to feed yawl-controller, that is the key.

I have one at my workbench feeding a 24v FCOB rgbw strip. Neat effects but each "pixel" is a bit chunky. Much better suited to larger space ambient lighting.

**Comment by chunkysteveo** [comment]
> This also works fine for 24v strips, as long as you have a source of 5v to feed yawl-controller, that is the key.
> 
> I have one at my workbench feeding a 24v FCOB rgbw strip. Neat effects but each "pixel" is a bit chunky. Much better suited to larger space ambient lighting.

Yeah, these 12v ones share about 70mm of COB leds 'per pixel'... smooth gradients don't work too well, but Rainbow blends nicely though 👌🏻
